drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -492,7 +503,8 @@ enum wifi_nrf_status tx_cmd_prep_callbk_fn(void *callbk_data,
 	phy_addr = wifi_nrf_hal_buf_map_tx(fmac_dev_ctx->hal_dev_ctx,
 					   nwb_data,
 					   buf_len,
-					   desc_id);
+					   desc_id,
+					   config->num_tx_pkts);
```

Comment:
the prototype says `frame_id`, but we are passing `config->num_tx_pkts` ?

Reply:
`num_tx_pkts` is a misnomer, its incremented for each packets, so, its an index.